generators/authentication/templates/config/passport.js

```diff
@@ -151,6 +151,7 @@ module.exports = {
      */
     onPassportAuth(req, res, error, user, info) {
       if (error || !user) return res.negotiate(error || info);
+      if (info.name === 'TokenExpiredError') info.status = 401; return res.negotiate(info)
```

Comment:
i havent tested it but i'm pretty sure it sets the http status to 401 without the `info.status = 401` it wont set the right status and will still return 500..